code/modules/clothing/clothing_accessories.dm

```diff
@@ -1,21 +1,20 @@
 /obj/item/clothing/proc/can_attach_accessory(obj/item/clothing/accessory/A)
-	if(valid_accessory_slots && istype(A) && (A.slot in valid_accessory_slots))
-		if(accessories.len && restricted_accessory_slots && (A.slot in restricted_accessory_slots))
+	var/obj/item/clothing/accessory/attach = A
```

Comment:
Why are we assigning `A` to `attach` when it is the same type, and neither var is assigned to again?  Couldn't we just use `A` throughout the proc with the same effect?

Reply:
For some reason it seemed to go horribly wrong without this define. I'm basically just a monkey, here, I'm not sure why it was failing.